client/src/node_api/mqtt/mod.rs

```diff
@@ -31,14 +28,14 @@ use crate::{Client, NetworkInfo, Result};
 impl Client {
     /// Returns a handle to the MQTT topics manager.
     #[cfg(feature = "mqtt")]
-    pub fn subscriber(&mut self) -> MqttManager<'_> {
+    pub fn subscriber(&self) -> MqttManager<'_> {
```

Comment:
These mut removals are technically breaking change so I would at least mention them in the changelog.